src/app/courses-page/edit-course-page/edit-course-page.component.html

```diff
@@ -0,0 +1,44 @@
+<form class="root">
```

Comment:
you can create one component just for management course and two components per two pages "new" and "edit" - this will help to avoid duplication